chcemvediet/apps/accounts/migrations/0005_socialapp_remove_twitter_and_linkedin.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.contrib.sites.models import Site
+from django.db import models, migrations
+
+
+def forward(apps, schema_editor):
+    SocialApp = apps.get_model(u'socialaccount', u'SocialApp')
+    SocialAccount = apps.get_model(u'socialaccount', u'SocialAccount')
+    SocialApp.objects.filter(provider__in=[u'twitter', 'linkedin']).delete()
+    SocialAccount.objects.filter(provider__in=[u'twitter', u'linkedin']).delete()
+
+def backward(apps, schema_editor):
+    SocialApp = apps.get_model(u'socialaccount', u'SocialApp')
+    site = Site.objects.get_current()
```

Comment:
@viliambalaz Pozor! Model `Site` nemozes v migraciach importovat napriamo. Iba cez `apps.get_model` ako vsetky ostatne modely.

Reply:
Aha, chapem, ked ziskas `Site` classu pomocou `apps.get_model`, tak vtedy nema `.get_current()` metodu. Na druhe zamyslenie teda asi naozaj to site ID musime mat v migracii uvedene explicitne.

Mame ho uvedene explicitne aj v `chcemvediet/settings/common.py` v konstante `SITE_ID`. Tak zrejme ho treba vsade uvadzat takto explicitne. Skoda.

Pozor, v migracii nesmieme importnut ani `chcemvediet/settings/common.py`, kedze settingy sa casom menia a migracia sa nesmie po nasadeni nikdy menit. Cize najzrozumitelnejsie bude asi priamo v migracii definovat konstantu `SITE_ID = 1` a potom pouzivat explicitne `pk=SITE_ID`, ako si to mal povodne.